app/core/config/application.js

```diff
@@ -1,35 +1,11 @@
 // @flow
 
-import {
-  API_KEY_GOOGLE_MAPS,
-  AUTH_ANDROID_GOOGLE_CLIENTID,
-  AUTH_IOS_GOOGLE_CLIENTID,
-  AUTH_KIWI_BACKEND,
-  SENTRY_DSN,
-} from 'react-native-dotenv';
+import { SENTRY_DSN } from 'react-native-dotenv';
 
 const application = {
   animations: {
     duration: 100, // ms
   },
-  apiKey: {
-    googleMaps: sanitize(API_KEY_GOOGLE_MAPS),
```

Comment:
This used to be used in MMB, as far as I know.
We don't use it at all in hotels?

Reply:
This is in `secrets.xml`

Reply:
> We don't use it at all in hotels?

Yes, but we get it from native app. If we need to use it in development, we add it to `secrets.xml`

Reply:
Okay, but these only work for Android. It's fine with me since MMD required to make API calls to Google's API if I remember well.